docs/_rules/xUnit2005.md

```diff
@@ -5,37 +5,47 @@ category: Assertions
 severity: Warning
 ---
 
-# This is a documentation stub
-
-Please submit a PR with updates to the [appropriate file]({{ site.github.repository_url }}/tree/master/docs/{{ page.relative_path }}) or create an [issue](https://github.com/xunit/xunit/issues) if you see this.
-
 ## Cause
 
-A concise-as-possible description of when this rule is violated. If there's a lot to explain, begin with "A violation of this rule occurs when..."
+A violation of this rule occurs when comparing two value type objects using `Assert.Same` or `Assert.NotSame`.
```

Comment:
nit: `int` would be a value type. `5` would still be a value type object (or value type value?!). I wanted to avoid making this sound like actual types are being compared.